website/docs/reference/transforms/dedupe.md.erb

```diff
@@ -0,0 +1,66 @@
+<%- component = metadata.transforms.dedupe -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+<%- if component.requirements.any? -%>
+## Requirements
+
+<%= component_requirements(component) %>
+
+<%- end -%>
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list, heading_depth: 3) %>
+
+<%- end -%>
+<%- if component.output.log -%>
+## Output
+
+<%= component_output(component, component.output.log, heading_depth: 3) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+<%= component_sections(component) %>
+
+### Cache Behavior
+
+This transform is backed by an LRU cache of size `cache.num_entries`.
```

Comment:
Please update this to use `num_events` (see above)

Reply:
done